src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,42 @@
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import React, { useContext } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoContext } from '../../context/TodoContext';
+import { TodoItem } from '../TodoItem';
+import { TempTodo } from '../TempTodo/TempTodo';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.